agent/profile

```diff
@@ -1,35 +1,44 @@
 # -*- mode: shell-script -*-
-prefix=/opt/pbench-agent
 
-if [[ ! -f ${prefix}/profile ]]
-then
-    if [[ -f ${PWD}/profile ]]
+function _pbench_pathins() {
+    if [ -d "$1" ] && [[ ":${PATH:-}:" != *"$1:"* ]] ; then
```

Comment:
Seems to be missing a leading `:`.

Reply:
I'd like to have a pointer to the pbench root visible outside, but if it's a real problem I can pull that one out.